Source/Core/Core/HW/EXI/EXIBrawlback.cpp

```diff
@@ -31,6 +32,7 @@ T swap_endian(T in)
 void writeToFile(std::string filename, uint8_t* ptr, size_t len)
 {
   std::ofstream fp;
+  std::filesystem::create_directory(SConfig::GetInstance().m_brawlbackReplayDir + "/" + SConfig::GetInstance().m_details_game_id);
```

Comment:
This writeToFile function should just write to file.

before calling this function, create the directory if needed.